.gitignore

```diff
@@ -11,3 +11,6 @@ npm-debug.log
 
 # Local configurations
 .vscode
+
+# Ignore generated MDX docs
+/src/design-tokens/generated/*.mdx
```

Comment:
Previously this ignore was in a different .gitignore in that folder. But, that did not work for Prettier + ESLint because they are set to only ignore files that are ignored in _this_ file